kubernetes/kubernetes/src/main/java/org/arquillian/cube/kubernetes/impl/DefaultConfiguration.java

```diff
@@ -447,4 +447,86 @@ public boolean isTrustCerts() {
     protected void setToken(String token) {
         this.token = token;
     }
+
+    @Override
+    public String toString() {
+
+        String SEP = System.getProperty("line.separator");
```

Comment:
There is a method called `System.lineSepartor()` that returns the same. Also name the variable `lineSeparator` so it does not confuse a reader to think that this is a const